### PR TITLE
Add login, logout commands to CLI remote --help

### PR DESCRIPTION
**Pull request by Vathsan**
Fixes https://github.com/wso2/micro-integrator/issues/1035

### CODE REVIEW

File: cmd/cmd/remote.go [hunk 1]
```diff
@@ -26,8 +26,8 @@ import (
 )
 
 const remoteCmdLiteral = "remote"
-const remoteCmdShortDesc = "Add, remove, update or select Micro Integrator"
-const remoteCmdLongDesc = "Add, remove, update or select Micro Integrator which will be associated with the CLI\n"
+const remoteCmdShortDesc = "Add, remove, update or select Micro Integrator, login or logout of Micro Integrator"
```
Comment (sdkottegoda) [inline cmd/cmd/remote.go]: How about, "Add, login to, logout of, remove, update or select Micro Integrator"

File: cmd/cmd/remote.go [hunk 2]
```diff
@@ -39,6 +39,8 @@ Available Commands:
   update [nick-name] [host] [port]         Update a Micro Integrator
   select [nick-name]                       Select a Micro Integrator on which commands are executed
   show                                     Show available Micro Integrators
+  login                                    Login to the current Micro Integrator instance
```
Comment (sdkottegoda) [inline cmd/cmd/remote.go]: "Login to the selected Micro Integrator"?

File: cmd/cmd/remote.go (hunk 1 above)
Comment (Vathsan) [reply cmd/cmd/remote.go]: Fixed in 2514a25d799cfe90b35dd0fdda17773b127eb19c

File: cmd/cmd/remote.go (hunk 2 above)
Comment (Vathsan) [reply cmd/cmd/remote.go]: Fixed in 2514a25d799cfe90b35dd0fdda17773b127eb19c

**Pull request merged**